foundation/grid.css

```diff
@@ -0,0 +1,22 @@
+/* GRID */
+.grid {
+    display: grid;
+    grid-gap: 20px;
+    grid-template-columns: repeat(auto-fit, minmax(50px, 1fr));
```

Comment:
I know this is a foundation, but i don't like to have in the same class that declares grid a column template set.

How about adding a class especially for that column alignment.

Reply:
The grid isn't part of the foundation, I only use it to build the front layout